modules/veteran/app/models/veteran/service/organization.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Veteran
+  module Service
```

Comment:
To me the term service isn't a great one to use, just because it's double meaning

Reply:
I'm trying to be literal to what the VA calls it, VSO or Veteran Service Organization or Veteran Service Representative (instead of Officer) makes the most sense and when you think that in the future we can merge say Veteran Verification or other Veteran related data into a module like this then having the Service namespace in place I think is prudent.

Reply:
I get what you're saying I just think the extra module + the fact that service is an overloaded word just due to the nature of it's use in programming classes it doesn't make for a good look.

Willing to be wrong here though. Maybe we should get a third opinion

Reply:
I totally understand that Service means something different for us in terms of software and development then in the way we are using it at the VA and I'm willing to be wrong in my nesting it as well.

@dhall or @charleystran would love your opinion here on nesting modules to separate their concerns from other future Veteran related models.

Reply:
yea, I agree with both sides. In any other case I would say dont use service, but in this case use service but have a very descriptive comment pointing out that it is not a logical service.

Reply:
I added a comment about this not being an actual service object to be on the safe side.